docs/source/tuning.rst

```diff
@@ -125,6 +125,60 @@ Experimenting with this value yields a better result at ``2e2``.
 Here, the scan matcher used rotation to still slightly mess up the result though.
 Setting the ``rotation_weight`` to ``4e2`` leaves us with a reasonable result.
 
+Special Cases
+-------------
+
+The default configuration and the above tuning steps are focused on quality.
+Only after we have achieved good quality, we can further consider low latency tasks and other special cases.
+
+Low Latency
+^^^^^^^^^^^
+
+Low latency is required for online algorithms, such as robot localization.
+Local SLAM, which operates in the foreground, directly affects latency.
+Global SLAM builds up a queue of background tasks. When global SLAM cannot keep up the the queue,
+drift can accumulate indefinitely, so global SLAM should be tuned to work in real time.
+
+To tune the different components for speed, there are many options, and we list them ordered from 
```

Comment:
switch sentence around: There are many options to..., and we

Reply:
Done